backend/yandex/yandex_crawl.py

```diff
@@ -0,0 +1,49 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+def yandex_crawler(max_pic_num):
+    """To get images from Yandex when search for skin cancer dermoscopy.
+    Parameters
+        max_pic_num : int.
+    Returns:
+        A list of dictionaries.
+        Each dictionary has
+            - Image source: The image address
+            - Host: "Yandex"
+            - Original URL: URL to the web page where the image is found
+            - Title: The title of the image
+    """
+    url = "https://yandex.com/images/search?text=skin%20cancer%20dermoscopy"
+    source_code = requests.get(url)
+    plain_text = source_code.text
+    soup = BeautifulSoup(plain_text, "html.parser")
+    counter = 1
+    data_list = []
+    for link in soup.findAll('a', {"class": "serp-item__link"}):
+        # get link
+        href = link.get('href')
+        yandex_link = "https://yandex.com/" + href
+        # getimage link
+        string = href.split("img_url=")[1]
+        convert = string.replace("%2F", "/")
+        convert2 = convert.replace("%3A", ":")
+        og_link = convert2.split('&text')[0]
+        # adding dict
+        data = {}
+        data['Image source'] = og_link
+        data['Host'] = "Yandex"
+        data['Origin URL'] = yandex_link
```

Comment:
The hosts are not Yandex. Yandex is just the search engine. The original URL should point to the page where the the image is found, too. For example:
![image](https://user-images.githubusercontent.com/19296926/100303758-df898700-2fcf-11eb-8d53-22dcace21108.png)


Here the host is `betterfamilyhealth.org`, the title is `Dermatology Treatments for Skin & other Medical Conditions`, and the original URL should be `http://www.betterfamilyhealth.org/dermatology-treatments.html` (the link that Yandex redirect to)

Reply:
Currently the original redirect to Yandex Image homepage.

Reply:
Can you help me with this ? Because:
+First thing first, the more I test my code, the more the site tries to block me, and I have to wait for a long time before the I can test my code again ( Yes, I have use all the alternate gg colab I have)
+ Secondly, All the element I was able to take comes from the main searching page:
![Untitled](https://user-images.githubusercontent.com/59214466/100321970-1bcfde00-2ff6-11eb-93b7-f01a69f977cb.png)
and as far as I try, all the elements on the detail image (what is show on your image) is protected and bs4 is unable to capture it. Furthermore, I try to scrape on the main page like before but it's not available either.

Reply:
I see, I'll take a look.

Reply:
Apparently that result is generated by JavaScript so it wasn't generated. We could probably use Selenium for interaction, but due to this time limit, we won't handle this.

Please let the url be `None` then. Also resolve other change requests.